AppLovin MAX Demo App - Java/app/src/main/java/com/applovin/enterprise/apps/demoapp/ads/applovin/SplashActivity.java

```diff
@@ -0,0 +1,58 @@
+package com.applovin.enterprise.apps.demoapp.ads.applovin;
```

Comment:
This Activity is redundant to the one [here](https://github.com/AppLovin/AppLovin-MAX-SDK-Android/blob/master/AppLovin%20MAX%20Demo%20App%20-%20Java/app/src/main/java/com/applovin/enterprise/apps/demoapp/SplashActivity.java) . Please remove.

Reply:
@santoshbagadi Can you confirm if this means - remove SplashActivity altogether for max demo app?

Reply:
No, just remove the duplicate one please. Keep the existing one.